includes/class-wc-payments-woopay-button-handler.php

```diff
@@ -126,7 +126,7 @@ public function init() {
 		// Create WooPay button location option if it doesn't exist and enable all locations by default.
 		if ( ! array_key_exists( self::BUTTON_LOCATIONS, get_option( 'woocommerce_woocommerce_payments_settings' ) ) ) {
 			if ( isset( $this->gateway->form_fields[ self::BUTTON_LOCATIONS ]['options'] ) ) {
-				$all_locations = $this->gateway->form_fields[ self::BUTTON_LOCATIONS ]['options'];
+				$all_locations = $this->gateway->get_form_fields()[ self::BUTTON_LOCATIONS ]['options'];
```

Comment:
Since `$this->gateway->form_fields` wasn't available, we should probably use `->get_form_fields()` in the `isset` as well, what do you think? Or we might want to remove the `isset`?

Reply:
Nice catch! I missed it when resolving the conflicts. Since we now have history of failure in this piece, I think it's better to keep the `isset`. Updated it in 7c24b63.